engines/scumm/imuse_digi/dimuse.cpp

```diff
@@ -52,6 +52,8 @@ IMuseDigital::IMuseDigital(ScummEngine_v7 *scumm, Audio::Mixer *mixer, int fps)
 	_sound = new ImuseDigiSndMgr(_vm);
 	assert(_sound);
 	_callbackFps = fps;
+	if (_vm->_game.id == GID_FT)
+		_callbackFps *= 2;
```

Comment:
This is the change I'm referring to in the PR description.

Reply:
I now have a proper explaination to support why this change above is technically closer to the real thing (while I'm doing the whole decomp thing). @aquadran 

So it appears, from the decomp, that there is a 100 Hz fine grained timer, under which, **fades** and **triggers** are handled at 60 Hz, and simple volume/panning changes (and sound switching I think) are handled at 10 Hz.
With the change above, **everything in the iMUSE engine is handled at 20 Hz** (10*2), which is a good compromise for now, since at the moment the engine is not able to handle the three timers (and the three correspondent behaviors) separatedly with the current code structure.

Reply:
After further study I must make some corrections about what I said above: 
while fades and triggers are handled at 60 Hz, it appears that the only thing operated at 10 Hz is the gain reduction on music while speech is playing.

The rest of it (sounds, volume changes, and crossfades generated by a JUMP instruction) are handled at 100 Hz.

I don't think currently the engine can handle filling the sound buffer at 100 fps speed (it can do it, but there are going to be some hiccups here and there). Still, the system needs to be speedier than it currently is, at least 60Hz for fades and triggers detection.

Gain reduction will have to stay at 10Hz, like the game code says.

I'll investigate further.